README.md

```diff
@@ -26,7 +26,7 @@ _ "github.com/influxdata/telegraf/plugins/outputs/warp10"
 
 ```
 [[outputs.warp10]]
-warpUrl = "https://warp1.cityzendata.net/api/v0/update"
+warpUrl = "https://warp.senx.io/api/v0/update"
```

Comment:
Here we could put a Warp 10 standalone as localhost URL instead of a distributed URL

Reply:
Changed